src/devices/README.md

```diff
@@ -205,6 +205,10 @@ Our vision: the majority of .NET bindings are written completely in .NET languag
 
 * [SPI, GPIO and I2C drivers for FT4222](Ft4222/README.md)
 
+### USB PD Power Sinks
```

Comment:
this is not necessary. Add a category.txt file to the root of the director with this category and it will be build automatically.